kotlin/samples/todo-android/common/src/main/java/com/squareup/sample/todo/TodoListsWorkflow.kt

```diff
@@ -20,19 +20,14 @@ import com.squareup.workflow.Sink
 import com.squareup.workflow.StatelessWorkflow
 import com.squareup.workflow.makeEventSink
 
-data class TodoListsRendering(
-  val lists: List<TodoList>,
-  val onRowClicked: (Int) -> Unit
-)
-
-class TodoListsWorkflow : StatelessWorkflow<List<TodoList>, Int, TodoListsRendering>() {
+class TodoListsWorkflow : StatelessWorkflow<List<TodoList>, Int, TodoListsScreen>() {
```

Comment:
Note that this workflow is able to remain unaware of the current selection. The parent workflow adds that information to this child's rendering while assembling the composite rendering.

Reply:
Probably worth adding a comment to that effect in the sample source code.